modules/wowchemy/assets/js/wowchemy.js

```diff
@@ -32,10 +32,11 @@ function removeQueryParamsFromUrl() {
  * --------------------------------------------------------------------------- */
 
 function toggleSearchDialog() {
-  if ($('body').hasClass('searching')) {
+  const body = document.querySelector('body');
+  if (body.classList.contains('searching')) {
     // Clear search query and hide search modal.
-    $('[id=search-query]').blur();
-    $('body').removeClass('searching compensate-for-scrollbar');
+    document.querySelector('[id=search-query]').blur();
```

Comment:
let's change this to be consistent with how you referred to the ID on line 67: `document.getElementById('search-query')`